wrappers/aws/aws-sdk-go/aws/dynamodb_factories.go

```diff
@@ -171,6 +216,89 @@ func handleDynamoDBScan(
 	}
 }
 
+func updateWithJsonMap(
+	destination map[string]string,
+	destinationKey string,
+	data map[string]interface{},
+) error {
+	stream, err := json.Marshal(data)
+	if err != nil {
+		return err
+	}
+	destination[destinationKey] = string(stream)
+	return nil
```

Comment:
Can we return ok, err (even though ok is ignored and just error checked)? 
Otherwise the code that calls this looks a bit confusing (if response != nill then return, instead of if response.err != nill)

Reply:
Changed it to return a boolean only (the error is ignored anyway)